applications/FluidDynamicsApplication/tests/FluidRVETest/fluid_rve_test_parameters.json

```
@@ -0,0 +1,68 @@
+{
+    "problem_data": {
+        "problem_name": "fluid_rve_test",
+        "parallel_type": "OpenMP",
+        "echo_level": 0,
+        "start_time": 0.0,
+        "end_time": 1.0
```

```suggestion
        "end_time": 0.1
```
To reduce the test runtime.

---

Changed.